tests/contacts/test_remote_sync.py

```diff
@@ -97,7 +97,7 @@ def test_auth_error_handling(contact_sync, default_account, db):
     db.session.commit()
 
     contact_sync.start()
-    contact_sync.join(timeout=10)
+    contact_sync.join(timeout=15)
```

Comment:
it this needed still? It looks like the test would be skipped

Reply:
Right, I added a separate issue about [removing it](https://github.com/closeio/sync-engine/issues/316).